go/mysql/constants.go

```diff
@@ -440,6 +440,9 @@ const (
 	// SSHandshakeError is ER_HANDSHAKE_ERROR
 	SSHandshakeError = "08S01"
 
+	// SSServerShutdown is ER_SERVER_SHUTDOWN
+	SSServerShutdown = "08S01"
```

Comment:
this is the same # as above. is that right?

Reply:
Yeah, and there is one more above it. I'm not sure what does it mean, but that's state from the reference manual for that type of error.